plugins/sql/plugin.spec.yaml

```
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: sql
 title: SQL
 description: The SQL plugin allows you to run SQL queries
-version: 3.0.2
+version: 3.0.3
+supported_versions: ["MySQL", "MSSQL", "PostgresSQL"]
```

These should contain version numbers of each of these SQL products. For example:
```suggestion
supported_versions: ["MySQL 5.7.0 - 5.7.35", "MSSQL 15", "PostgresSQL 10.0 - 13.0"]
```
Please replace version numbers with actual versions used and tested. Does not have to be a range, can just be one or two versions of each.